scripts/install-docker.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+export DEBIAN_FRONTEND=noninteractive
+export TZ=America/Los_Angeles
+
+locale -a
+locale-gen en_US.UTF-8
+export LC_ALL=en_US.UTF-8
+export LANG=en_US.UTF-8
+
+apt-get update -qq && apt-get upgrade -qq
+apt-get install -y -qq \
```

Comment:
Are these dependencies necessary for Docker installation?

Reply:
Unlikely, I can try to trim it down if it's important.  I would guess these are handy to have around; it's copied from @krfricke's example. @krfricke do you know which ones are okay to remove?

Reply:
It would be beneficial to determine which dependencies are necessary and which are not. If merging this pull request is not time-sensitive, we can allocate some time to investigate this further.

Reply:
Makes sense -- as far as timing is concerned we would just need it before the 0.6.0 release, since this issue depends on it: https://github.com/ray-project/kuberay/issues/1199

Reply:
Fixed fd0b35e